src/fake_keyboard.cpp

```diff
@@ -325,6 +325,112 @@ void fake_keyboard::ydo_type_char(uint32_t c) {
     }
 }
 
+// Send Ctrl+V (paste) to the active window using the configured method
+void fake_keyboard::send_ctrl_v() {
+    // Delay to allow clipboard to update
+    std::this_thread::sleep_for(std::chrono::milliseconds(200));
+
+#ifdef USE_X11_FEATURES
+    switch (m_method) {
+        case method_t::ydo:
+            send_ctrl_v_ydo();
+            break;
+        case method_t::legacy:
+            send_ctrl_v_legacy();
+            break;
+        case method_t::xdo:
+            send_ctrl_v_xdo();
+            break;
+    }
+#else
+    // Non-X11 build: use ydo path (Wayland) by default
+    if (m_ydo_daemon_socket >= 0) {
+        send_ctrl_v_ydo();
+    } else {
+        LOGW("no method available to send ctrl+v");
+    }
+#endif
+}
+
+// Helper implementations split to avoid code duplication for the ydo path.
+// ydo implementation is usable on both X11 and Wayland builds.
+void fake_keyboard::send_ctrl_v_ydo() {
+    // helper closure: emit a key event and wait a small fixed duration
+    auto press_and_wait = [&](uint16_t code, int32_t val) {
+        ydo_uinput_emit(EV_KEY, code, val, 1);
+        std::this_thread::sleep_for(std::chrono::duration<int, std::milli>(
+            settings::instance()->fake_keyboard_delay()));
+    };
+
+    press_and_wait(KEY_LEFTCTRL, 1);
+    press_and_wait(KEY_V, 1);
+
+    std::chrono::milliseconds(200ms);
```

Comment:
```suggestion
    std::this_thread::sleep_for(200ms);
```

Most likely, you wanted this

Reply:
You are correct, indeed.